generator/django/template/appInformation.json

```diff
@@ -4,20 +4,26 @@
             "callbackUrl": "http://localhost:8000/callback"
         },
         "metadata": {
-            "name": "Affinidi Login integration using Django and Authlib",
-            "description": "A reference app created using django framework and uses AuthLib ibrary to enable passwordless login using Affinidi IDP ",
-            "idp": "Affinidi",
+            "name": "Affinidi Login Integration using Django and Authlib",
+            "description": "A code sample using Django framework and uses AuthLib library to enable passwordless login using Affinidi IDP.",
+            "idp": {
+                "name": "Affinidi",
+                "link": "https://www.affinidi.com/product/affinidi-login"
+            },
             "framework": {
                 "name": "Django",
-                "iconUrl": "https://static.djangoproject.com/img/logos/django-logo-negative.svg"
+                "iconUrl": "django-logo.svg",
+                "link": "https://www.djangoproject.com/"
             },
             "language": {
                 "name": "Python",
-                "iconUrl": "https://s3.dualstack.us-east-2.amazonaws.com/pythondotorg-assets/media/community/logos/python-logo-only.png"
+                "iconUrl": "python-logo.svg",
```

Comment:
Is it iconUrl or iconName?

Reply:
I've changed it to `icon` to align with the rest of the properties.